spec/requests/maintenance_spec.rb

```diff
@@ -58,7 +58,7 @@
         end
 
         it 'redirects sign_in if the token is wrong' do
-          get('/users/sign_in', {maintenance_token: "#{token}3"})
+          get('/users/sign_in', params: {maintenance_token: "#{token}3"})
```

Comment:
What’s the dangling “3” after the token?

Reply:
I honestly, don't know.